pkg/graph/schema.resolvers.go

```diff
@@ -2118,6 +2204,15 @@ func (r *queryResolver) CedarPersonsByCommonName(ctx context.Context, commonName
 	return response, nil
 }
 
+// CedarSoftwareProducts is the resolver for the cedarSoftwareProducts field.
+func (r *queryResolver) CedarSoftwareProducts(ctx context.Context, cedarSystemID string) (*models.CedarSoftwareProducts, error) {
+	cedarSoftwareProducts, err := r.cedarCoreClient.GetSoftwareProductsBySystem(ctx, cedarSystemID)
+	if err != nil {
+		return nil, err
+	}
+	return cedarSoftwareProducts, nil
```

Comment:
nit: if preferable, we can just

```suggestion
	return r.cedarCoreClient.GetSoftwareProductsBySystem(ctx, cedarSystemID)
```

Reply:
per our call - let's leave as is for now! thanks @downeyn-cms!!

Reply:
For context - the reason we are leaving it for now is that most of the other resolver handlers seem to have the same unnecessary explicit nil check.

Rather than refactor this and have it not match the existing pattern, we are going to leave it as is and make a tech debt ticket to change this pattern across the entire schema resolver. I will make the ticket and link it here and to the JIRA story.